scripts/deployments/RunWorkflows.ps1

```diff
@@ -179,14 +179,14 @@ Param(
   [string]$EnvironmentName="",
   [string]$GitHubRepo=$null,
   [string]$GitHubRef=$null,
-  [string]$LoginInteractiveTenantId=$null,
+  [string]$LoginInteractiveTenentId=$null,
   [SecureString]$LoginServicePrincipalJson=$null,
   [string]$WorkingDirectory=(Resolve-Path "../.."),
   [SecureString]$NvaUsername=$null,
   [SecureString]$NvaPassword=$null
 )
 
-#Requires -Modules Az, powershell-yaml
+# Requires -Modules Az, powershell-yaml
```

Comment:
#requires is a keyword in PowerShell. It’s not a comment. 

See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_requires